python/ray/rllib/utils/actors.py

```diff
@@ -36,7 +36,11 @@ def completed_prefetch(self):
 
         for worker, obj_id in self.completed():
             plasma_id = ray.pyarrow.plasma.ObjectID(obj_id.id())
-            ray.worker.global_worker.plasma_client.fetch([plasma_id])
+            if not ray.global_state.use_raylet:
+                ray.worker.global_worker.plasma_client.fetch([plasma_id])
+            else:
+                ray.worker.global_worker.local_scheduler_client\
```

Comment:
minor, but let's avoid using `\` and do this with parentheses, e.g., something like

```python
(ray.worker.global_worker.local_scheduler_client
    .reconstruct_objects([obj_id], True))
```

or whatever `yapf` insists on. 

From https://www.python.org/dev/peps/pep-0008/

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.